### PR TITLE
Add feature to upload jmx script and test data file

### PR DESCRIPTION
**Pull request by pranaysahith**

### CODE REVIEW

File: jmeter-icap-poc/scripts/delete_stack.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import boto3
 from datetime import timedelta, datetime, timezone
-import argparse
 from create_stack import Config
 
+
```
Comment (jpoolGW) [inline jmeter-icap-poc/scripts/delete_stack.py:5]: Remove unneeded white space

File: jmeter-icap-poc/scripts/delete_stack.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import boto3
 from datetime import timedelta, datetime, timezone
-import argparse
```
Comment (jpoolGW) [inline jmeter-icap-poc/scripts/delete_stack.py:3]: Should this be removed? I Know Ali re-added this to his scripts yesterday

**Review by jpoolGW — CHANGES_REQUESTED** [review]

File: jmeter-icap-poc/scripts/delete_stack.py (hunk 1 above)
Comment (pranaysahith) [reply jmeter-icap-poc/scripts/delete_stack.py]: Updated.

File: jmeter-icap-poc/scripts/delete_stack.py (hunk 2 above)
Comment (pranaysahith) [reply jmeter-icap-poc/scripts/delete_stack.py:3]: `import argparse` is not needed anymore. its not being used anywhere in the script, so I removed it.

**Review by jpoolGW — APPROVED** [review]

**Pull request merged**